frontend/src/components/Sidebar/Sidebar.tsx

```diff
@@ -69,19 +69,19 @@ export default function Sidebar() {
   const { t, i18n } = useTranslation(['glossary', 'frequent']);
   const specialSidebarOptions: SidebarItemProps[] = [
     {
-      name: t('frequent|back'),
+      name: 'back',
       icon: 'mdi:hexagon-multiple',
       label: t('glossary|Cluster'),
```

Comment:
I wonder if you checked if the label has the same problem?

Reply:
The label doesn't have the same problem because that's the user-facing text and it should be translated.
We could rename this item as `clusters` instead of `back` though. Let me do that.